README.md

```diff
@@ -58,11 +58,11 @@ You will then need to use [bind mounts](https://docs.docker.com/storage/bind-mou
 
 ```ShellSession
 docker pull quay.io/kubespray/kubespray:v2.17.1
-docker run --rm -it --mount type=bind,source="$(pwd)"/inventory/sample,dst=/inventory \
+docker run --rm -it --mount type=bind,source="$(pwd)"/inventory/,dst=/kubespray/inventory/ \
```

Comment:
Actually this command is modeled based on the assumption that you `git clone`'ed the kubespray repo and our sample inventory vars are in `inventory/sample`.

Reply:
I'd like to be able to modify the local inventory after `git clone` and be able to run it through docker.

Reply:
So I think that all the inventory must be mounted to the image.